### PR TITLE
update to 1.9

### PR DESCRIPTION
**Pull request by melindaloubser1**
Move e2e tests to expected location in 1.9
Update requirements & dockerfile

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM rasa/rasa-sdk:1.8.1
+FROM rasa/rasa-sdk:1.9.0
```
Comment (TyDunn) [inline Dockerfile]: Neesd to be rasa[spacy]~=1.9.1

Comment (melindaloubser1) [reply Dockerfile]: In the requirements though right? Not the dockerfile

Comment (TyDunn) [reply Dockerfile]: yes! just wanted to comment it somewhere before you merged, since I had accepted

Comment (melindaloubser1) [reply Dockerfile]: sure, I changed it, I don't think you approved though?

**Pull request merged**